packages/next/client/image.tsx

```diff
@@ -676,15 +676,21 @@ export default function Image({
       }
     }
   }
-
   const imgStyle = Object.assign({}, style, layout === 'raw' ? {} : layoutStyle)
+  const svgBlurPlaceholder = `url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns=\'http%3A//www.w3.org/2000/svg\' xmlns%3Axlink=\'http%3A//www.w3.org/1999/xlink\' viewBox=\'0 0 1280 853\'%3E%3Cfilter id=\'b\' color-interpolation-filters=\'sRGB\'%3E%3CfeGaussianBlur stdDeviation=\'50\'%3E%3C/feGaussianBlur%3E%3CfeComponentTransfer%3E%3CfeFuncA type=\'discrete\' tableValues=\'1 1\'%3E%3C/feFuncA%3E%3C/feComponentTransfer%3E%3C/filter%3E%3Cimage filter=\'url(%23b)\' x=\'0\' y=\'0\' height=\'100%25\' width=\'100%25\' xlink%3Ahref=\'${blurDataURL}\'%3E%3C/image%3E%3C/svg%3E");`
```

Comment:
You should set the viewBox to the `width` and `height` of the image we are applying this to.

Reply:
Makes sense, done